src/locale/lang/fr.js

```diff
@@ -0,0 +1,85 @@
+export default {
+  el: {
+    datepicker: {
+      now: 'Maintenant',
+      today: "Aujourd'hui",
```

Comment:
I see. So could you please replace these words with their appropriate abbreviations? Thanks.